docs/source/config.rst

```diff
@@ -226,6 +226,16 @@ should be absolute with respect to the workload rootfs. Files will be placed
 together in the output directory. You cannot specify the directory structure of
 the output.
 
+rootfs-size
+^^^^^^^^^^^^^^^^^
+The desired rootfs size (in human-readable units, e.g. "4GB"). This number must
```

Comment:
The docs imply decimal units ("GB"), but `humanfriendly.format_size()` is called with `binary=True`.  The test JSON uses IEC binary units ("GiB").  Maybe this should be clarified?

Reply:
I can make the example and the docs consistent, but it actually doesn't matter. humanfriendly .parse_size reads things correctly (GiB will be binary and GB will be decimal). It always prints in binary units and the suffix will indicate that (format_size is only for logging purposes). Users are free to do whatever they please.